package.json

```diff
@@ -37,6 +37,13 @@
   },
   "prettier": "eslint-config-3box/prettier.config",
   "dependencies": {
+    "@opentelemetry/api": "^1.1.0",
+    "@opentelemetry/api-metrics": "^0.28.0",
+    "@opentelemetry/core": "^1.2.0",
+    "@opentelemetry/exporter-metrics-otlp-http": "^0.28.0",
```

Comment:
Not used

Reply:
hm I thought I had run `npm uninstall -save` but maybe I had not - removed!